spec/hash-map.dd

```diff
@@ -430,55 +430,65 @@ $(SPEC_RUNNABLE_EXAMPLE_RUN
         ---
         )
 
-$(H2 $(LNAME2 properties, Properties))
+$(H2 $(LNAME2 properties, Properties and Operations))
 
-$(P Properties for associative arrays are:)
+$(P Properties and operations for associative arrays are:)
 
     $(TABLE_2COLS Associative Array Properties,
-        $(THEAD Property, Description)
-        $(TROW $(D .sizeof), Returns the size of the reference to the associative
-        array; it is 4 in 32-bit builds and 8 on 64-bit builds.)
-        $(TROW $(D .length), $(ARGS Returns number of values in the
+        $(THEAD Property/Operation, Description)
+        $(TROW $(D sizeof), $(ARGS The size of the reference to the associative
+        array; it is 4 in 32-bit builds and 8 on 64-bit builds.))
+        $(TROW $(D empty), $(ARGS $(D true) if the associative array has no elements; otherwise $(D false).))
```

Comment:
`empty` is not a built-in property, it's defined here:
https://dlang.org/phobos/std_range_primitives.html#empty